modyn/config/schema/pipeline/evaluation/handler.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+from typing import List
+
+from modyn.config.schema.base_model import ModynBaseModel
+from pydantic import Field, model_validator
+
+from .strategy import EvalStrategyConfig, UntilNextTriggerEvalStrategyConfig
+from .trigger import AfterTrainingEvalTriggerConfig, EvalTriggerConfig
+
+
+class EvalHandlerConfig(ModynBaseModel):
+    name: str = Field(
+        "Modyn EvalHandler", description="The name of the evaluation handler used to identify its outputs in the log"
+    )
+    strategy: EvalStrategyConfig = Field(description="Defining the strategy and data range to be evaluated on.")
+    trigger: EvalTriggerConfig = Field(None, description="Configures when the evaluation should be performed.")
+    datasets: List[str] = Field(
+        description="All datasets on which the model is evaluated.",
+        min_length=1,
```

Comment:
I don't get why we have another datasets list here. We already have a datasets list in the evaluation config. Why do we need to specify the IDs again?